mtags/src/main/scala-3/scala/meta/internal/pc/PcCollector.scala

```diff
@@ -667,3 +673,29 @@ case class ExtensionParamOccurence(
     sym: Symbol,
     methods: List[untpd.Tree],
 )
+
+case class EndMarker(symbol: Symbol)
+
+object EndMarker:
+  val endMarkerRegex = """.*end(/\*.*\*/|\s)+""".r
```

Comment:
```suggestion
  private val endMarkerRegex = """.*end(/\*.*\*/|\s)+""".r
```
and maybe an example of what it matches?